lib/src/package.dart

```diff
@@ -139,66 +139,64 @@ class Package {
-    Directory outPolymerDirectory = new Directory(outPolymerPath);
-    Directory outDartdocViewerDirectory = new Directory(outDartdocViewerPath);
-
-    outWebComponentsDirectory.renameSync(webWebComponentsPath);
-    outPolymerDirectory.renameSync(webPolymerPath);
-    outDartdocViewerDirectory.renameSync(webDartdocViewerPath);
-  }
+  // /**
+  //  * Moves the packages folder into the root of the web folder.
+  //  *
+  //  * WARNING: this may change in the future versions dartdoc-viewer.
+  //  */
+  // void moveDocumentationPackages(Version version) {
+  //   String out = join(BUILD_DOCUMENTATION_ROOT_PATH, "${name}-${version}",
+  //       DARTDOC_VIEWER_OUT);
+  //   String webPath = join(out, 'web');
+  //   String webPackagesPath = join(webPath, 'packages');
+  //   String outPackagesPath = join(out, 'packages');
+  //
+  //   // 1) remove symlink in out/web/packages
+  //   Directory webPackagesDirectory = new Directory(webPackagesPath);
+  //   webPackagesDirectory.deleteSync();
+  //
+  //   // 2) only copy dartdoc_viewer specific packages
+  //   _moveDartDocViewerSpecificFiles(outPackagesPath, webPackagesPath);
+  // }
+
+  // void _moveDartDocViewerSpecificFiles(String outPackagesPath, String webPackagesPath) {
+  //   // mkdir web/packages
+  //   // copy -r packages/web_components web/packages/
+  //   // copy -r packages/polymer web/packages/
+  //
+  //   Directory webPackagesDirectory = new Directory(webPackagesPath);
+  //   webPackagesDirectory.createSync();
+  //
+  //   String outWebComponentsPath = join(outPackagesPath, "web_components");
+  //   String outPolymerPath = join(outPackagesPath, "polymer");
+  //   String outDartdocViewerPath = join(outPackagesPath, "dartdoc_viewer");
+  //
+  //   String webWebComponentsPath = join(webPackagesPath, "web_components");
+  //   String webPolymerPath = join(webPackagesPath, "polymer");
+  //   String webDartdocViewerPath = join(webPackagesPath, "dartdoc_viewer");
+  //
+  //   Directory outWebComponentsDirectory = new Directory(outWebComponentsPath);
+  //   Directory outPolymerDirectory = new Directory(outPolymerPath);
+  //   Directory outDartdocViewerDirectory = new Directory(outDartdocViewerPath);
+  //
+  //   outWebComponentsDirectory.renameSync(webWebComponentsPath);
+  //   outPolymerDirectory.renameSync(webPolymerPath);
+  //   outDartdocViewerDirectory.renameSync(webDartdocViewerPath);
+  // }
 
   /**
    * Builds documentation for a particular version of a package.
    */
   int buildDocumentationSync(Version version, String dartSdkPath, {bool verbose: false}) {
     String outputFolder = 'docs';
-    String packagesFolder = './packages'; // The pub installed packages
+    //String packagesFolder = './packages'; // The pub installed packages
     String workingDirectory = join(BUILD_DOCUMENTATION_ROOT_PATH,
         "${name}-${version}");
 
-    List<String> args = ['--compile',
-                         '--no-include-sdk',
```

Comment:
dartdoc doesn't generate or link to the sdk (or other package source) by default. It _can_ optionally generate separate docs for the sdk.